Stack/Opc.Ua.Core/Stack/Server/ServerBase.cs

```diff
@@ -1537,6 +1537,14 @@ public void ScheduleIncomingRequest(IEndpointIncomingRequest request)
                         Utils.LogTrace("Too many operations. Total: {0} Active: {1}",
```

Comment:
in the past discussion there was mentioned BadTooManyOperations is not the right error code. Lets check what other options there are.

Reply:
conclusion --> new statuscode BadServerTooBusy can be used